src/main/java/edu/harvard/iq/dataverse/ConfigureFragmentBean.java

```diff
@@ -55,9 +51,9 @@
     
     public String configureExternalAlert() {
         generateApiToken();
+        PrimeFaces.current().executeScript("location.reload(true)");
```

Comment:
@lubitchv could you briefly explain what this change does?

Reply:
It reloads the dataverse page. Without reloading the page one would not see the notification. There is also another reason for that. If user has several files in DataSet and he uses external tool for the first file and later returns for the page to use external tool for the second file, the fileId will remain of the first file end external tool will show the user content of the first file and not of a second. It confuses users tremendously. We had an experience that many of users do not pay attention to the message that asks them to reload the browser. So I thought it is better to do for them automagically.

Reply:
@lubitchv OK, we'll have @kcondon test that and see how it works. Can you also then remove the message from the bundle (I'm assuming, of course, it was only used in this one place)?